lib/flutter_typeahead.dart

```diff
@@ -669,6 +672,7 @@ class _TypeAheadFieldState<T> extends State<TypeAheadField<T>>
   void dispose() {
     this._suggestionsBoxController.widgetMounted = false;
     WidgetsBinding.instance.removeObserver(this);
+    this._resizeOnScrollTimer?.cancel();
```

Comment:
Do we need this this. (for both _resize and _suggestionsBoxController) above?  I am pretty sure the Google Linting rules suggests to not use this. unless really needed... ??

Reply:
I don't usually write the "this" but I saw it was used on the two lines above. That's why I used the "this".

Reply:
I updated the code, removing the "this."